moodler/download.py

```diff
@@ -109,3 +135,108 @@ def download_grading_worksheet(assignment_id,
         grading_worksheet_file.write(response.content)
 
     return grading_worksheet_file_name
+
+
+def download_course_grades_report(course_id,
```

Comment:
Just remove the other grade export functionality, and integrate this one in the moodle's export functions